### PR TITLE
Multiple UI Improvements

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys, have a look. This PR includes changes from the following tickets:

* MM-1819 - Adding for user name in Private messages.
* MM-1820 - Adding tags for the channel intro text.
* MM-1654 - Adding max height to channel description popover and allowing scroll.
* MM-1939 - Adjusting style of unread posts above bar.
* MM-1831 - Making channel names bold for modals
* MM-1839 - Showing preview footer when user is hovering over the left/right arrows for previews

### CODE REVIEW

**Comment by crspeller** [comment]
@asaadmahmoodspin When editing jsx files in the future you will have to meet the style guide found here: https://github.com/mattermost/platform/blob/master/STYLE-GUIDE.md You don't have to refactor like the rest of the team does, but new code should follow it. Let me know if you need any help getting ESLint setup.

**Comment by crspeller** [comment]
Code looks good.

**Comment by hmhealey** [comment]
Looks good to me

File: web/react/components/view_image.jsx [hunk 1]
```diff
@@ -109,6 +109,22 @@ module.exports = React.createClass({
             }
         );
 
+        $(this.refs.previewArrowLeft.getDOMNode()).hover(
```
Comment (hmhealey) [inline web/react/components/view_image.jsx]: Looking back over this now that this pull request was reverted, I'm pretty certain it's this chunk that caused the UI to break. The arrows only exist if there is more than one file attached to the post so this.refs.previewArrowLeft/this.refs.previewArrowRight would be undefined when a post only had one attachment.

**Pull request merged**